compliance_checker/suite.py

```diff
@@ -359,6 +360,9 @@ def run(self, ds, skip_checks, *checker_names):
                     vals.extend(self._run_check(c, ds, max_level))
                 except Exception as e:
                     errs[c.__func__.__name__] = (e, sys.exc_info()[2])
+            if isinstance(checker, CFBaseCheck):
+                for res in checker.appendix_a_results:
+                    vals.append(res)
```

Comment:
I don't understand why this needs to be here? Why couldn't `CFBaseCheck` (or it subclass) include the appendix A checks in its output? That way `suite.py` could remain independent of any particular check suite, which I thought was the intention.

Reply:
Will move to a `check_` method instead.